### DISCUSSION TITLE
Reindex and delivery of filters lasts more than 2 days

### DISCUSSION THREAD

**Discussion by gmichel-brandung** [scayle/community#4] [Q&A] [answered]
Hey guys. 

Does anyone know how to accelerate the delivery of filters?
We have setup some filter for a specific category in a "fresh" shop via SCAYLE panel and they are still not requestable via the api.

I am thankful for any hint

**Comment by HenriBeck** [comment]
Hey, normally these changes can take up to one hour, and filters are only shown if there are products with the attribute groups assigned to them.

Please open a technical support request via our service desk if there are products with the attribute group in the category.

**Reply by gmichel-brandung** [reply]
@HenriBeck Hey, thank you for the fast answer. we have been waiting for 2 days and they are still not requestable.
We will make a support request.

**Reply by HenriBeck** [reply, marked as answer]
Yeah only saw the time in the title after writing the comment.

This shouldnt take 2 hours but there might also be other reasons why a filter is not returned
